Composer/packages/ui-plugins/virtual-assistant-creation/src/components/provisionSummaryPage.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+
+import { RouterPaths } from '../shared/constants';
+
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+
+// Content Data
+const provisioningDetailItems = () => {
+  return [
+    {
+      key: '0',
+      Resource: formatMessage('Azure Bot Service'),
+      Notes: formatMessage(
+        'The Azure Bot Service resource stores configuration information that allows your Virtual Assistant to be accessed on the supported Channels and provide OAuth authentication.'
+      ),
+    },
+    {
+      key: '1',
+      Resource: formatMessage('Azure Blob Storage'),
+      Notes: formatMessage('Used to store conversation transcripts.'),
+    },
+    {
+      key: '2',
+      Resource: formatMessage('Azure Cosmos DB'),
+      Notes: formatMessage('Used to store conversation state.'),
+    },
+    {
+      key: '3',
+      Resource: formatMessage('Azure App Service Plan'),
+      Notes: formatMessage('Used to host your Bot Web App and QnA Maker Web App.'),
+    },
+    {
+      key: '4',
+      Resource: formatMessage('Azure Application Insights'),
+      Notes: formatMessage('Used to capture conversation and application telemetry.'),
+    },
+    {
+      key: '5',
+      Resource: formatMessage('Bot Web App'),
+      Notes: formatMessage('Hosts your Bot application.'),
+    },
+    {
+      key: '6',
+      Resource: formatMessage('Language Understanding'),
+      Notes: formatMessage('Subscription keys for Language Understanding Cognitive Service.'),
+    },
+    {
+      key: '7',
+      Resource: formatMessage('QnA Maker'),
+      Notes: formatMessage(
+        'Subscription keys for QnA Maker Cognitive Service which facilitates the bot personality you selected.'
+      ),
+    },
+    {
+      key: '8',
+      Resource: formatMessage('QnA Maker Web App'),
+      Notes: formatMessage('Hosts your QnA Maker knowledgebases'),
+    },
+    {
+      key: '9',
+      Resource: formatMessage('QnA Maker Azure Search Service'),
+      Notes: formatMessage('Search index for your QnA Maker knowledgebases.'),
+    },
+  ];
+};
+
+const columns: IColumn[] = [
+  {
+    key: 'column1',
+    name: formatMessage('Resource'),
```

Comment:
Here is a case where columns is a const value rather than a function, but it calls formatMessage.  We should be consistent depending on what Ben Y says above.

Reply:
Okay I have changed above to const, will wait on what Ben says to determine if they should stay that way.

Reply:
See my comment above.